auth0/resource_auth0_custom_domain_verification.go

```diff
@@ -52,14 +62,20 @@ func createCustomDomainVerification(ctx context.Context, d *schema.ResourceData,
 
 		d.SetId(customDomainVerification.GetID())
 
-		if result := readCustomDomainVerification(ctx, d, m); result.HasError() {
-			return resource.NonRetryableError(fmt.Errorf("failed to read custom domain verification"))
+		// The cname_api_key field is only given once: when verification
+		// succeeds for the first time. Therefore, we set it on the resource in
+		// the creation routine only, and never touch it again.
+		if err := d.Set("cname_api_key", customDomainVerification.CNAMEAPIKey); err != nil {
```

Comment:
Does this need to be wrapped in `d.Partial(true)`? I can't really figure out what the intent of that API is, but it would be _nice_ to not lose this key if the subsequent read fails for some transient reason.

Reply:
Hey @impl 👋🏻  That's correct, there's no need for a `d.Partial(true)` over here. It's mostly a legacy function that was used together with `d.SetPartial("field")`, but honestly it was having a very weird behavior most of the times and considering terraform always refreshes before each operation it was really not needed.